### PR TITLE
(Better to get a new branch for this) Testing script based on v2

### PR DESCRIPTION
**Pull request by theo1701**
An automated script for testing the behaviour of the machine learning model from the four variables: learning rate, number of hidden layers, nodes each layer, and iterations. 

Added a run param for setting the learning rate parameter, with the default as tf.keras's Adam optimizer's default value.

The test can be run by the commands

`python3 init_test.py`
`python3 test.py`

For compactness, the test directories and json files are not included in the git files. Some might need to be manually created. There should be a test_plan.json in the same level as the py scripts, a _test_auto.json in configs/run, and a test_results/omniTest.json.

The programs supports "discrete runs", in that the testing process can be shut down manually. As long as the test required json files are not altered, the program will automatically commence tests starting from the test that was not finished when it shut down.

I don't have permission to create a new branch so it is set to merge into v2 right now.

That should be the all that's important about this test setup. -insert happy face here-

### CODE REVIEW

**Comment by theo1701** [comment]
closing since it causes conflicts and does not support the new settings (lrscheduler)
